lib/modules/txt.mjs

```diff
@@ -343,3 +343,28 @@ export async function validateCopyrightNoticeSectionIsNumbered (doc, { mode = MO
 
   return result
 }
+
+/**
+ * Validate separated formfeeds
+ *
+ * @param {Object} doc Document to validate
+ * @param {Object} [opts] Additional options
+ * @param {number} [opts.mode=0] Validation mode to use
+ * @returns {Array} List of errors/warnings/comments or empty if fully valid
+ */
+export async function validateSeparatedFormfeeds (doc, { mode = MODES.NORMAL } = {}) {
+  const result = []
+
+  if (mode === MODES.SUBMISSION) return result
+
+  if (doc.data.pageCount !== doc.data.contains.pagesFound) {
+    result.push(new ValidationWarning(
+      'PAGES_NOT_SEPARATED_BY_FORMFEEDS',
+      'Pages are not separated by formfeeds.',
+      {
+        ref: 'https://trustee.ietf.org/wp-content/uploads/IETF-TLP-4.pdf'
```

Comment:
The TLP doesn't speak about formfeeds. I think  https://authors.ietf.org/en/drafting-in-plaintext is the only thing we can point to (though it doesn't explicitly call out _require_ FF - that comes in in passive voice in https://datatracker.ietf.org/doc/rfc7994/.)